site/src/load.rs

```diff
@@ -199,6 +222,16 @@ impl SiteCtxt {
             all_commits,
         )
     }
+
+    /// Download master-branch Rust commits if the cached value is older than one minute
+    pub async fn update_master_commits(&self) -> anyhow::Result<()> {
```

Comment:
I would prefer that we make this method return the cached result, rather than needing to call update() and then load the result from the field.